get-started/06_bind_mounts.md

```diff
@@ -25,8 +25,8 @@ volume drivers are available to support other use cases ([SFTP](https://github.c
 | - | ------------- | ----------- |
 | Host Location | Docker chooses | You control |
 | Mount Example (using `-v`) | my-volume:/usr/local/data | /path/to/data:/usr/local/data |
-| Populates new volume with container contents | Yes | No |
-| Supports Volume Drivers | Yes | No |
+| Populates new volume with container contents | No | Yes |
+| Supports Volume Drivers | No | Yes |
```

Comment:
Hm.. this doesn't look correct; only (named) volumes support volume drivers, and only volumes will copy data from the container to the volume (if it's empty); bind-mounts on the other hand take the files that are present on the host, and make them available to the container (but won't copy files that were present inside the container to the bind-mounted directory).